tests/db/schemas/mssql.sql

```diff
@@ -179,7 +179,7 @@ CREATE TABLE params (
 
 CREATE TABLE tags (
 	key VARCHAR(250) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
-	value VARCHAR(5000) COLLATE "SQL_Latin1_General_CP1_CI_AS",
+	value VARCHAR(8000) COLLATE "SQL_Latin1_General_CP1_CI_AS",
```

Comment:
not sure why this changed, i think maybe we missed out on running `./tests/db/update_schemas.sh` in #13176?

Reply:
I ran `update_schemas.sh`:

https://github.com/mlflow/mlflow/actions/runs/11790377345/job/32840716342?pr=13740

It looks like we only check `tests/resources/db/latest_schema.sql`.

Reply:
Filed https://github.com/mlflow/mlflow/pull/13761